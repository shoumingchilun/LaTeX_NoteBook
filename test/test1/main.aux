\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</P(\376\377\0000)>>}
\citation{ref1}
\citation{ref2}
\HyPL@Entry{1<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}NoSQL的起源}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}NoSQL数据库的发展原因}{1}{section.2}\protected@file@percent }
\citation{ref3}
\@writefile{toc}{\contentsline {section}{\numberline {3}NoSQL技术简介}{3}{section.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}NoSQL概念与特性}{3}{subsection.3.1}\protected@file@percent }
\citation{ref4}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}基于Key值的数据模型}{4}{subsection.3.2}\protected@file@percent }
\citation{ref5}
\citation{ref6}
\citation{ref6}
\@writefile{toc}{\contentsline {section}{\numberline {4}MongoDB简介}{6}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}MongoDB特性}{6}{subsection.4.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}MongoDB应用场景}{9}{subsection.4.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.3}MongoDB学习心得}{11}{subsection.4.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {5}其他常见的NoSQL数据库实现}{13}{section.5}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1}文档型数据库}{13}{subsection.5.1}\protected@file@percent }
\citation{ref8}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.1}CouchDB}{14}{subsubsection.5.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.2}OrientDB}{14}{subsubsection.5.1.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.3}Terrastore}{15}{subsubsection.5.1.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.2}时序数据库}{16}{subsection.5.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.2.1}Apache Cassandra}{16}{subsubsection.5.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.3}其他开源数据库}{17}{subsection.5.3}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.3.1}Hypertable}{17}{subsubsection.5.3.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.3.2}Redis}{17}{subsubsection.5.3.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.3.3}HBase}{18}{subsubsection.5.3.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {6}总结}{18}{section.6}\protected@file@percent }
\bibcite{ref1}{1}
\bibcite{ref2}{2}
\bibcite{ref3}{3}
\bibcite{ref4}{4}
\bibcite{ref5}{5}
\bibcite{ref6}{6}
\bibcite{ref7}{7}
\bibcite{ref8}{8}
\gdef \@abspage@last{21}
