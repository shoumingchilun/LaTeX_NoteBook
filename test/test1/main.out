\BOOKMARK [1][-]{section.1}{\376\377\000N\000o\000S\000Q\000L\166\204\215\167\156\220}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000N\000o\000S\000Q\000L\145\160\143\156\136\223\166\204\123\321\134\125\123\237\126\340}{}% 2
\BOOKMARK [1][-]{section.3}{\376\377\000N\000o\000S\000Q\000L\142\200\147\057\173\200\116\313}{}% 3
\BOOKMARK [2][-]{subsection.3.1}{\376\377\000N\000o\000S\000Q\000L\151\202\137\365\116\016\162\171\140\047}{section.3}% 4
\BOOKMARK [2][-]{subsection.3.2}{\376\377\127\372\116\216\000K\000e\000y\120\074\166\204\145\160\143\156\152\041\127\213}{section.3}% 5
\BOOKMARK [1][-]{section.4}{\376\377\000M\000o\000n\000g\000o\000D\000B\173\200\116\313}{}% 6
\BOOKMARK [2][-]{subsection.4.1}{\376\377\000M\000o\000n\000g\000o\000D\000B\162\171\140\047}{section.4}% 7
\BOOKMARK [2][-]{subsection.4.2}{\376\377\000M\000o\000n\000g\000o\000D\000B\136\224\165\050\127\072\146\157}{section.4}% 8
\BOOKMARK [2][-]{subsection.4.3}{\376\377\000M\000o\000n\000g\000o\000D\000B\133\146\116\140\137\303\137\227}{section.4}% 9
\BOOKMARK [1][-]{section.5}{\376\377\121\166\116\326\136\070\211\301\166\204\000N\000o\000S\000Q\000L\145\160\143\156\136\223\133\236\163\260}{}% 10
\BOOKMARK [2][-]{subsection.5.1}{\376\377\145\207\150\143\127\213\145\160\143\156\136\223}{section.5}% 11
\BOOKMARK [3][-]{subsubsection.5.1.1}{\376\377\000C\000o\000u\000c\000h\000D\000B}{subsection.5.1}% 12
\BOOKMARK [3][-]{subsubsection.5.1.2}{\376\377\000O\000r\000i\000e\000n\000t\000D\000B}{subsection.5.1}% 13
\BOOKMARK [3][-]{subsubsection.5.1.3}{\376\377\000T\000e\000r\000r\000a\000s\000t\000o\000r\000e}{subsection.5.1}% 14
\BOOKMARK [2][-]{subsection.5.2}{\376\377\145\366\136\217\145\160\143\156\136\223}{section.5}% 15
\BOOKMARK [3][-]{subsubsection.5.2.1}{\376\377\000A\000p\000a\000c\000h\000e\000\040\000C\000a\000s\000s\000a\000n\000d\000r\000a}{subsection.5.2}% 16
\BOOKMARK [2][-]{subsection.5.3}{\376\377\121\166\116\326\137\000\156\220\145\160\143\156\136\223}{section.5}% 17
\BOOKMARK [3][-]{subsubsection.5.3.1}{\376\377\000H\000y\000p\000e\000r\000t\000a\000b\000l\000e}{subsection.5.3}% 18
\BOOKMARK [3][-]{subsubsection.5.3.2}{\376\377\000R\000e\000d\000i\000s}{subsection.5.3}% 19
\BOOKMARK [3][-]{subsubsection.5.3.3}{\376\377\000H\000B\000a\000s\000e}{subsection.5.3}% 20
\BOOKMARK [1][-]{section.6}{\376\377\140\073\176\323}{}% 21
